tests/test_e2e_40_sdntrace.py

```diff
@@ -942,3 +942,50 @@ def test_080_validate_attribute_on_payload(self):
         api_url = KYTOS_API + '/amlight/sdntrace_cp/v1/traces'
         response = requests.put(api_url, json=payload)
         assert response.status_code == 200
+
+    def test_085_test_evcs_terminating_on_nnis(cls):
+        "Test EVCs terminating on NNIs"
+
+        cls.create_evc(999, "00:00:00:00:00:00:00:02:1", "00:00:00:00:00:00:00:04:1")
+        
+        payload = {
```

Comment:
`pw_s3` looks good, I didn't get why initially `pw_s2` had to be created though, unless you planned to also trace it but didn't? Let me know what you meant.

Reply:
Thank you, @viniarck. 
It was reproducing the example of Italo in [issue 102](https://github.com/kytos-ng/sdntrace_cp/issues/102). Anyway, you are right, so I removed `pw_s2`.